test/test_cartesian_graph.py

```diff
@@ -383,7 +383,7 @@ def test_to_sympy():
         )
 
 
-def test_catch_invalid_sympy_expr():
+def test_handle_sympy_expr_with_infinities():
```

Comment:
Perhaps a more descriptive name. With your change, infitinies are *not* handled, right?

Reply:
good point. tried something new.